motioneye/rootfs/etc/cont-init.d/motioneye.sh

```diff
@@ -45,3 +40,24 @@ if ! bashio::fs.directory_exists '/share/motioneye'; then
     mkdir -p /share/motioneye \
         || bashio::exit.nok 'Failed to create initial motionEye media folder'
 fi
+
+# Remove any existing action buttons before recreating
+for old_action in lock unlock light alarm up right down left zoom preset; do
+  if ls /data/motioneye/${old_action}* > /dev/null 2>&1; then
+	  rm /data/motioneye/${old_action}*
+  fi
+done
+
+# Creates action button scripts if any are configured
+if bashio::config.has_value 'action_buttons'; then
+    bashio::log.info "Configuring action buttons."
+    for button in $(bashio::config "action_buttons|keys"); do
+        BUTTON_TYPE=$(bashio::config "action_buttons[${button}].type")
+	CAMERA_NUMBER=$(bashio::config "action_buttons[${button}].camera")
+	BUTTON_COMMAND=$(bashio::config "action_buttons[${button}].command")
+        bashio::log.debug "File: ${BUTTON_TYPE}_${CAMERA_NUMBER}, Command: ${BUTTON_COMMAND}"
+	echo "#!/bin/bash" > "/data/motioneye/${BUTTON_TYPE}_${CAMERA_NUMBER}"
```

Comment:
Multiple `echo`'s to a single file can be combined using a single redirect.
e.g.:
```bash
{
  echo "Line 1"
  echo "Line 2"
} > file
```

Reply:
Thanks for the fast review, I think I have covered all the needed fixes.